pages/orgs/[orgId]/upgrade.tsx

```diff
@@ -3,16 +3,32 @@ import { useRouter } from "next/router";
 import styles from "../../../styles/Home.module.css";
 
 import { Heading, Skeleton, useToast } from "@chakra-ui/react";
+import { useEffect } from "react";
 import { useProducts } from "../../../api/stripe/useProducts";
 import ProductCard from "../../../components/ProductCard";
+import { parseBooleanEnvValue } from "../../../config/parser/parseBooleanEnvValue";
+import Routes from "../../../routes/routes";
 
 export default function EditOrgPage() {
   const router = useRouter();
   const toast = useToast();
-  const { products, isError, isLoading } = useProducts();
 
   const orgId = Number(router.query.orgId);
 
+  useEffect(() => {
+    if (!router.isReady) return;
+
+    function navigateToDashboardPage() {
+      router.push(Routes.DASHBOARD);
+    }
+
+    if (!parseBooleanEnvValue(window.__env.NEXT_PUBLIC_STRIPE_ENABLED)) {
```

Comment:
Why do you access the window.__env directly instead of using loadConfig which uses it to configure the configuration values?